src/main/java/utils/LoginUtils.java

```diff
@@ -0,0 +1,26 @@
+package utils;
+
+import dao.UserDao;
+import daoImpl.UserDaoImpl;
+import models.User;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+public class LoginUtils {
+    public static boolean login(HttpServletRequest request) {
+
+        String login = request.getParameter("login");
+        String password = request.getParameter("password");
+
+        UserDao dao = UserDaoImpl.getInstance();
+        User user = dao.login(login, password);
```

Comment:
This is a business logic. It should be moved to a business-logic layer. For example to the UserService. It's a good practice to keep such logic in a middle layer.